astrodynamics-fall-2018/astrodynamicsFall2018PDD.tex

```diff
@@ -0,0 +1,372 @@
+      % That way you can choose key information from the sections that you've already completed and string them together in the abstract.
+      % Consider the abstract to be your elevator pitch to anyone reading this design document.
+      % What are they reading?
+      % What is the goal?
+      % Why is it worth my time?
+      % The abstract is what will show up in Google results and other search engines, and what people will read when they are deciding what is worth their time and brain power.
+\end{abstract}
+
+\label{sec:nomenclature}
+\newcommand{\nomunit}[1]{%
+\renewcommand{\nomentryend}{\hspace*{\fill}#1}}
+\renewcommand{\nompreamble}{
+    % If you include mathematical expressions or express variables in the design document, list them with their corresponding definitions here as a list.
+    % The two lines below make it look nice when defining units/values to constants.
+
+    % Note that math terms and non-math terms are separated and alphabetized, regardless of the order in which they are defined. (Recall terms \$like this\$ are in the math environment)
+    % Read more about advanced nomenclature formatting here:\\
+    % \url{https://www.sharelatex.com/learn/Nomenclatures}
+  }
+\nomenclature{RIT}{Rochester Institute of Technology}
+\nomenclature{SPEX}{RIT Space Exploration}
+\nomenclature{PDD}{Project Design Document}
+\nomenclature{CCD{}Charged Coupled Device}
+\nomenclature{MSD}{Multidisciplinary Senior Design}
+\nomenclature{SG}{Student Government}
+\printnomenclature{}
+
+
+% HELPFUL HINT: If you get the warning ``Command terminated with space.'' when using a \command try placing ``%'' or ``{}'' immediately following the command.
+
+% The sections included here are required. Additional sections and subsections may be added as necessary.
+\section{Introduction}
+\label{sec:introduction}
+  % The introduction is a place to give background and context before diving into the subject matter.
+  % Establish context for the work you are about to propose and the main ideas of the proposition itself.
+
+\IEEEPARstart{R}{IT} SPEX has direct ties to the RIT Observatory though one of the primary advisors and a mutual interest in astronomy.
+Despite this, both groups have had little collaboration.
+The RIT SPEX Astrodynamics team has some equipment that is measurable to the equipment in the observatory, but has trouble with operation and lack of experience.
+The RIT Observatory has a need for students to work on engineering projects, which RIT SPEX Astrodynamics can provide.
+RIT SPEX has always had members interested in observing using various telescopes, but these members have little training.
+There have already been projects that gave the team insight into the equipment behind observing.
+These include: building and modifying a rolling mount for its 12" Meade tele-scope, building mounts for an Orion sighting scope, building a barn door camera mount, and general maintenance on all telescopes.
```

Comment:
replace `"` (the single character containing double quote) with `''` (two single quotes) let latex do its magic
OR
say `12-inch`